app/src/main/java/edu/uc/haflga/financialpal/ui/main/MainViewModel.kt

```diff
@@ -4,4 +4,16 @@ import androidx.lifecycle.ViewModel
 
 class MainViewModel : ViewModel() {
     // TODO: Implement the ViewModel
+
+    private var moneyText = ""
+    private var result: FLoat = 0f
+
+    fun setAmount (value: String) {
+        this.moneyText = value
+        result = value.toFloat()
+    }
+    fun getResult(): Float? {
```

Comment:
You can probably remove the nullability operator here, ? ... I don't see any case where result would be null.